assignments/life_expectancy/cleaning.py

```diff
@@ -1,22 +1,8 @@
 import re
-import argparse
-from pathlib import Path
 import pandas as pd
 from pandas import DataFrame
-from life_expectancy.tests import OUTPUT_DIR
 from life_expectancy.region import Region
 
-
-def load_data(input_path: str|Path, delimiter: str = "[\t]") -> DataFrame:
-    """Function that load the .tsv data
-    :param input_path: path where the data to be loaded is located. Search in init.py to find OUTPUT_DIR. Raw data.
-    :param delimiter: pass the delimiter appropried when reading the input file
-    :return DataFrame: Loaded dataframe
-    """
-    return pd.read_csv(input_path, sep=delimiter, engine="python")
-
-
-
 def clean_data(df: DataFrame, region: Region = Region.PT) -> DataFrame:
     """Function that load the data
```

Comment:
clean data nao parece uma funçao que faça load the data. Acho que tens um docstring antigo